hls4ml/backends/vitis/passes/feature_check.py

```diff
@@ -0,0 +1,25 @@
+from hls4ml.model.optimizer import OptimizerPass
+
+
+class ValidateConvImplementation(OptimizerPass):
+
```

Comment:
Would prefer docstrings in new classes that are added describing their purpose.

Reply:
I'm planning a new PR that adds docstrings to most of the remaining codebase, not just Vitis parts.